src/components/frame/v5/pages/UserAdvancedPage/UserAdvancedPage.tsx

```diff
@@ -22,7 +22,7 @@ const MSG = defineMessages({
 const UserAdvancedPage = () => {
   const { user, userLoading, walletConnecting } = useAppContext();
 
-  useSetPageHeadingTitle(formatText({ id: 'userProfile.title' }));
+  useSetPageHeadingTitle(formatText({ id: 'advancedSettings.title' }));
```

Comment:
this part seems kinda confusing, I think it could be refactored some day 😄

Reply:
Yeah it really does at first glance 😅 Basically `userProfile.title` is now wrong because the page itself is the Advanced Page and not the Profile page